src/Symfony/Component/DependencyInjection/CHANGELOG.md

```diff
@@ -16,7 +16,8 @@ CHANGELOG
  * made `ServiceLocator` implement `ServiceProviderInterface`
  * deprecated support for non-string default env() parameters
  * added `%env(require:...)%` processor to `require()` a PHP file and use the value returned from it
-
+ * deprecated tag `tagged` in favour of `tagged_iterator`
```

Comment:
this will be for 4.4 now - please also update the UPGRADE files

Reply:
Done.